lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -154,11 +154,18 @@ def set_3d_properties(self, zs=0, zdir='z'):
         ys = self.get_ydata()
 
         try:
-            # If *zs* is a list or array, then this will fail and
-            # just proceed to juggle_axes().
-            zs = np.full_like(xs, fill_value=float(zs))
-        except TypeError:
-            pass
+            len_zs = len(zs)
+        except TypeError:  # object of type 'int' has no len()
```

Comment:
Perhaps better as:
```
zs = np.broadcast_to(zs, xs.shape)
```
Which does everything for you. If you wanted you could catch `ValueError` and print the shapes.